pyoptsparse/pyIPOPT/pyIPOPT.py

```diff
@@ -1,38 +1,26 @@
-# /bin/env python
 """
 pyIPOPT - A python wrapper to the core IPOPT compiled module.
 """
-# =============================================================================
-# IPOPT Library
-# =============================================================================
-
+# Compiled module
 try:
-    from . import pyipoptcore
+    from . import pyipoptcore  # isort: skip
```

Comment:
Can you explain why these have to be skipped?

Reply:
So there are two scenarios here.
1. For what you showed here, these are not necessary. If I don't skip them, then `isort` will just add a header here:
   ```
   -    from . import snopt  # isort: skip
   +    # Local modules
   +    from . import snopt
   ```
   This is a stylistic thing, and in the future I would like to move these imports inside the `__init__` which will eliminate this issue. However, if you think these are unnecessary we can definitely remove them.
2. There are other imports which are buried deep in the file. For example [this line](https://github.com/mdolab/pyoptsparse/blob/update-format/pyoptsparse/pyOpt_history.py#L220) imports the pyOptSparse version, and adding the skip prevents `isort` from adding an import header there which would be unnecessary.